### DISCUSSION TITLE
How do I add a folder containing multiple files into a GitHub repository?

### DISCUSSION THREAD

**Discussion by philliproach** [community/community#51283] [Repositories] [answered]
### Select Topic Area

General

### Body

I am trying to upload a folder containing multiple files and other folders to a GitHub repository but I cant find the option to upload folders to GitHub. I have looked everywhere and I have searched the web for other possible solutions but I cant find anything.

**Comment by yaramt** [comment, marked as answer]
1) Add file 
2) Upload files
3) Drag your folder and drop it to the specified area
4) Commit changes

**Reply by philliproach** [reply]
It works! Thank you so much
